ex01.py

```diff
@@ -0,0 +1,16 @@
+def flatten(x):
+    result = []
+    for element in x:
+        if hasattr(element, "__iter__") and not isinstance(element, str):
```

Comment:
hasattr to przesada, wystarczy sprawdzanie elementów przez isinstance

Reply:
uproszczono funkcje